### PR TITLE
Mines Act Permit v1.1.1 OCA

### PR DESCRIPTION
**Pull request by Jsyro**
Added subfolder with version to maintain access to older version OCA. 

`_code` attributes are now going to be their full strings.
`mine_party_appt` is now `permittee_name`

### CODE REVIEW

**Comment by swcurran** [comment]
What are you expecting the “live” version to be — the one in the `1.1.1` folder, or the files in the `mines-act-permit` folder, or both?  As the script to generate the root folder JSON files is currently desgined, it will not see or use the ones in the `1.1.1` folder.  Do we need to change that?

**Comment by Jsyro** [comment]
Maybe.... 

Shouldn't all published version of schemas have OCA bundles?  we have a 1.0 and a 1.1.1 version in production, those have slightly different attribute names so should have different OCA bundles.

**Comment by Jsyro** [comment]
I can quickly change the 0.3 (which was never pushed to production) as the subfolder one and put 1.1.1 in the root.

**Comment by amanji** [comment]
Technically shouldn't each version be a separate schema? So according to the README as written you'd have `OCABundles/schema/<versioned_schema>/`

**Comment by Jsyro** [comment]
We could do that, but in our example we have published the same schema on two different ledgers, and both are valid so the txn name as the folder doesn't really help. I'll make two top level folders. We are hoping to merge ASAP, we have a workshop at 10am this morning and i'd like to share the new OCA bundle ahead of time.

File: OCABundles/schema/bcgov-digital-trust/mines-act-permit-1_1_1/branding.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[
+    {
+        "logo": "https://raw.githubusercontent.com/bcgov/aries-oca-bundles/main/OCABundles/schema/bcgov-digital-trust/mines-act-permit/1.1.1/bc-logo.jpg",
```
Comment (swcurran) [inline OCABundles/schema/bcgov-digital-trust/mines-act-permit-1_1_1/branding.json:3]: Path is now wrong.

**Comment by swcurran** [comment]
@Jsyro — given the rush, how about I do another PR with this content?

**Comment by swcurran** [comment]
Replaced by #95

**Pull request closed without merging**